tsconfig.json

```diff
@@ -7,7 +7,15 @@
     "noEmit": true,
     "strict": true,
     "esModuleInterop": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "baseUrl": ".",
+    "paths": {
+      "*": ["*", "app/javascript/*"]
+    }
```

Comment:
What are these changes?

Reply:
Can you make that change?

Reply:
Also why would that be needed for glitch-soc, but not upstream Mastodon for `import * from 'mastodon/...'`?